content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst

```diff
@@ -0,0 +1,257 @@
+:guilabel:`Add to wave` to group the orders. In the resulting pop-up window, ensure the
+:guilabel:`Add to` field is set :guilabel:`a new [batch/wave] transfer`, and click
+:guilabel:`Confirm`. Doing so opens a batch or wave transfer with the orders included.
+
+.. figure:: dispatch/add-to-wave.png
+   :alt: Example wave.
+
+   Delivery orders are selected to be grouped into a wave transfer.
+
+Alternative batch creation method
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Another place to create batches is by going to the :menuselection:`Inventory` app, and in the
+:guilabel:`Delivery Orders` card, click the :icon:`fa-ellipsis-v` :guilabel:`(three dots)` icon.
+In the resulting drop-down menu, click :guilabel:`Prepare batch`.
+
+.. image:: dispatch/prepare-batch.png
+  :alt: Show prepare batch option from the new ... options.
+
+.. note::
+   In the :guilabel:`Transport Management` drop-down menu, these other options are used for the
+   following:
+
+   - :guilabel:`Manage Batches`: open list of batches.
+   - :guilabel:`Dock Dispatching`: open weekly calendar view of scheduled batch operations.
+   - :guilabel:`Batches by Route`: kanban view of batches grouped by fulfillment route.
+   - :guilabel:`Calendar`: open hourly calendar view of scheduled operations.
+   - :guilabel:`Statistics`: open pivot table of the batch transfers.
+
+.. _inventory/shipping_receiving/batch-form:
+
+Batch form
+----------
+
+On the batch transfer form, fill the following fields out accordingly:
+
+- :guilabel:`Responsible`: employee assigned to the picking. Leave this field blank if *any* worker
+  can fulfill this picking.
+- :guilabel:`Operation Type`: from the drop-down menu, select the operation type under which the
+  picking is categorized.
+- :guilabel:`Scheduled Date`: specifies the date by which the :guilabel:`Responsible` person should
+  complete the transfer to the output location.
+- :guilabel:`Dock Location`: select the loading location.
+- :guilabel:`Vehicle`: select the vehicle, which will auto-fill :guilabel:`Vehicle Category`.
+- :guilabel:`Vehicle Category`: configure the vehicle category to determine the vehicle's capacity.
+  Items in the batch, with the weight and volume fields configured on the product form, are totaled
+  to show how much space is remaining for loading products.
+
+The :guilabel:`Weight` and :guilabel:`Volume` fields display whether the items in the order exceed
+the :ref:`capacity set on the vehicle category <inventory/shipping_receiving/vehicle-capacity>`.
+
+.. example::
+   The :guilabel:`Volume` bar is grayed out because the capacity has been reached.
+
+   .. image:: dispatch/batch-form.png
+      :alt: Show batch form.
+
+Prepare delivery route
+~~~~~~~~~~~~~~~~~~~~~~
+
+Additional resources are available to prepare the driver for the delivery. On the top of the batch
+or wave form, click the :guilabel:`Map`. Doing so opens another page, displaying a map. Selecting an
```

Comment:
```suggestion
or wave form, click the :guilabel:`Map` button. Doing so opens another page, displaying a map. Selecting an
```

Reply:
Just a note- mine did not open another tab or page- it just went to the map. Might want to double-check and change accordingly.